src/CMakeLists.txt

```diff
@@ -14,10 +14,6 @@ add_library(${lib_name} STATIC ${fortran_src})
 add_library(${PROJECT_NAME}::${lib_name} ALIAS ${lib_name})
 
 set_target_properties(${lib_name} PROPERTIES Fortran_MODULE_DIRECTORY ${module_dir})
-if(OpenMP_Fortran_FOUND)
-  target_link_libraries(${lib_name} PRIVATE OpenMP::OpenMP_Fortran)
-endif()
-set_target_properties(${lib_name} PROPERTIES INTERFACE_LINK_LIBRARIES ${lib_name})
```

Comment:
remind me why we had this line in here?

Reply:
I don't remember. You don't need to set INTERFACE_LINK_LIBRARIES with yourself.